apps/disease_control/templates/disease_control/disease_and_condition_list_page.html

```diff
@@ -2,10 +2,10 @@
 {% load wagtailcore_tags %}
 
 {% block content %}
-{% include 'includes/breadcrumb.html' %}
+{% include 'includes/breadcrumb.html' with is_hidden=breadcrumb_is_hidden %}
```

Comment:
How about just not calling this `include` at all? So something like:
```
{% if not breadcrumb_is_hidden %}
  {% include 'includes/breadcrumb.html' %}
{% endif %}
```

Reply:
Done. Thanks